.ci/idea_inspection.sh

```diff
@@ -28,9 +28,6 @@ if [[ -z $IDEA_PATH ]]; then
     fi
 fi
 
-# Execute compilation of Checkstyle to generate all source files
-mvn -e --no-transfer-progress clean compile
-
```

Comment:
I have removed this from script since we ignore `target` directory during inspection anyway. This reduces execution time and makes execution in container locally easier.

Reply:
I think removal of this is why inspections are failing, testing now